webpack.config.js

```diff
@@ -4,7 +4,10 @@ const webpack = require('webpack');
 const path = require('path');
 
 module.exports = {
-    entry: './static/src/javascripts/boot-webpack.js',
+    entry: [
+        './static/src/javascripts/boot.js',
+        './static/src/javascripts/bootstraps/standard/main.js',
```

Comment:
I'm not 100% sure why this is necessary, but if you don't include `/bootstraps/standard/main.js` in the `entry` list, it ends up in a separate bundle. I'll do some reading around Webpack dynamic requires to see if we're missing something.